lib/repl.js

```diff
@@ -783,7 +783,7 @@ ArrayStream.prototype.writable = true;
 ArrayStream.prototype.resume = function() {};
 ArrayStream.prototype.write = function() {};
 
-const requireRE = /\brequire\s*\(['"](([\w\.\/-]+\/)?([\w\.\/-]*))/;
+const requireRE = /\brequire\s*\(['"](([\w./-]+\/)?([\w./-]*))/;
```

Comment:
@Fishrock123 No. we don't need to escape characters inside character class `[``]` except for `-` in non-tail position.